lib/cc/analyzer/container.rb

```diff
@@ -52,19 +52,14 @@ def run(options = [])
 
         command = docker_run_command(options)
         Analyzer.logger.debug("docker run: #{command.inspect}")
-        pid, _, out, err = POSIX::Spawn.popen4(*command)
+        _, out, err, @t_wait = Open3.popen3(*command)
```

Comment:
You may be right, we just want the status. Not sure if that would allow us to kill that process tough: https://github.com/codeclimate/codeclimate/blob/974c82b95e820b7d4b58df02e46b30019746ca98/lib/cc/analyzer/container.rb#L174